### PR TITLE
Add option for runtime caching

### LINKED ISSUE #1838: limit thumbnail cache to runtime
### Describe your suggested feature

I've made `@APPDATA@/files/coil` read only through root explorer to block thumbnail generation & the app became super snappy.

Thumbnail caching on storage media affects app performance a lot. Providing an option to limit thumb cache to runtime (memory), would be great. Because physical memory is always 10x faster than internal storage.

I'm sure thumb cache isn't necessary, as most users never revisit the older timeline multiple times. Even if they do, it wouldn't take much data & processing to reload those.

### Other details
I've researched a little bit about runtime mounts (tmpfs) on Android. But it's not possible although the Android file system is similar to Linux.

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Bnyro**
closes #1838

### CODE REVIEW

File: app/src/main/java/com/github/libretube/util/ImageHelper.kt [hunk 1]
```diff
@@ -25,17 +26,24 @@ object ImageHelper {
...
 
         imageLoader = ImageLoader.Builder(context)
             .callFactory(CronetHelper.callFactory)
-            .diskCache(diskCache)
+            .apply {
+                when (maxImageCacheSize) {
+                    "" -> {
+                        memoryCachePolicy(CachePolicy.DISABLED)
+                    }
+                    else -> diskCache(
```
Comment (FireMasterK) [inline app/src/main/java/com/github/libretube/util/ImageHelper.kt:39]: Would this mean caching is disabled by default?

Comment (Bnyro) [reply app/src/main/java/com/github/libretube/util/ImageHelper.kt:39]: Yes, is that a problem for the servers?
I'm always annoyed that the app size increases so much :/

Comment (FireMasterK) [reply app/src/main/java/com/github/libretube/util/ImageHelper.kt:39]: No, it's not a problem for servers but is a problem for app performance.
I think we should use both a memory and disk cache by default.

Let's say 20MB for disk, and 5MB of ram based caching.

**Comment by Bnyro** [comment]
Oh, my bad, I disabled memory cache instead of disk cache haha

**Comment by Bnyro** [comment]
Should be a oneliner though ...

**Pull request merged**